website/content/labs/lab7.md

```diff
@@ -180,33 +167,31 @@ Before continuing, make sure that you also push to your GitHub repository:
 $ git push
 ```
 
-That way, we can check that you've followed the steps described in this task.
+That way, we can check that you've followed the steps described in this task. You do not need to enter anything into Gradescope for this task.
 
 
-# Task 3: Create .travis.yml for cs220-helloapp-2019
+# Task 3: Create .travis.yml for HelloApp
 [40 points]
 
-As you remember from our [previous lab]({{< relref "lab6.md" >}}), we have a way to make sure our app passes all the tests every time we push.
+As you may remember from our [previous lab]({{< relref "lab6.md" >}}), we have a way to make sure our app passes all the tests every time we push.
 
-For this, you should create a `.travis.yml` file in your `cs220-helloapp-2019` repo. With python, there's no need to build, so you should just define a test phase which runs the command `pytest` as its test script.
+For this, you should create a `.travis.yml` file in your `2020-lab7-GITHUB_USERNAME` repo. With Python, there's no need to build, so you should just define a single job that runs the command `pytest` as its test script.
 
-In the last task, you made the tests fail. You should ensure the CI job fails when the tests are failing.
+In the last task, you made the tests fail. You should ensure the CI job fails when the tests are failing. Take the URL of the failed build, and enter it in Gradescope.
```

Comment:
It might be useful to mention that they should commit and push their `.travis.yml` file. Also, should we give them more information about the `os`, `dist`, etc., or leave them to figure out the file altogether?

Reply:
They're expected to figure out the file on their own, but I've clarified that they should be running `pytest` in a Linux environment. I've also added a reminder to commit and push their .travis.yml file.